classes/AutoPanMix.sc

```diff
@@ -94,7 +97,37 @@ AutoPanMix : BaseMix {
 		});
 
 		/*
-		* jacktrip_autopan_in is used to apply leveling, filters and panning to a specific client input, 
+		* jacktrip_simple_in is used to apply leveling and filters to a specific client input, 
+		* and send it to an audio bus
+		*
+		* \client : client number to use for bus & input channel offsets
+		* \lpf : frequency to use for low pass filter (default 20000)
+		* \hpf : frequency to use for high pass filter (default 20)
+		* \out : output bus to use for sending audio (default 0)
+		* \mul : amplitude level multiplier (default 1.0)
+		*/
+		"Sending SynthDef: jacktrip_simple_in".postln;
+		SynthDef("jacktrip_simple_in", {
+			var client = \client.ir(0);
+
+			// create an array containing all audio input channels for a specific client
+			var mix = Array.fill(2, { arg channelNum;
```

Comment:
Would it be better to not hardcode this `2` and make it `inputChannelsPerClient` instead?

Reply:
yes!